### PR TITLE
Feat/login backend

### PR DESCRIPTION
**Pull request by OropezaHugo**
create backend for login

### CODE REVIEW

File: backend/Controllers/LoginController.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        return Ok( await _loginService.PostToken(loginDto));
+    }
+
+    [HttpGet]
+    public async Task<IActionResult> GetCookie()
+    {
+        await _loginService.GetCookie();
+        return  Redirect("http://localhost:5173/personal-info");
+    }
+}
```
Comment (MayerliSantander) [inline backend/Controllers/LoginController.cs]: Add endline

File: backend/DTOs/WithoutId/LoginDTO.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+namespace DTOs.WithoutId;
+
+public class LoginDTO
+{
+    public string Email { get; set; }
+    public string Token { get; set; }
+}
```
Comment (MayerliSantander) [inline backend/DTOs/WithoutId/LoginDTO.cs]: Add endline

File: backend/DTOs/WithoutId/SessionDTO.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+namespace DTOs.WithoutId;
+
+public class SessionDTO
+{
+    public string Email { get; set; }
+    public string Token { get; set; }
+}
```
Comment (MayerliSantander) [inline backend/DTOs/WithoutId/SessionDTO.cs]: Add endline

File: backend/Services/LoginService.cs [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            HttpOnly = false, 
+            Secure = false,
+            Expires = DateTime.UtcNow.AddDays(1), // Expiración en 1 día
+            SameSite = SameSiteMode.None
+        };
+
+        _httpContextAccessor.HttpContext.Response.Cookies.Append("session", tokenHandler.WriteToken(token), cookieOptions);
+        return true;
+    }
+}
```
Comment (MayerliSantander) [inline backend/Services/LoginService.cs]: Add endline

File: backend/Services/ServicesInterfaces/ILoginService.cs [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+using Microsoft.AspNetCore.Mvc;
+
+namespace backend.Services.ServicesInterfaces;
+
+public interface ILoginService
+{
+    public Task<bool> PostToken(LoginDTO loginDto);
+    public Task<bool> GetCookie();
+    
+}
```
Comment (MayerliSantander) [inline backend/Services/ServicesInterfaces/ILoginService.cs]: Add endline

**Review by MayerliSantander — CHANGES_REQUESTED** [review]

**Review by DeidamiaFuentes — APPROVED** [review]

**Review by MayerliSantander — APPROVED** [review]

**Pull request merged**